enveloper/src/main/java/uk/gov/justice/services/event/enveloper/Enveloper.java

```diff
@@ -56,7 +57,7 @@ void register(@Observes final EventFoundEvent event) {
      * @return a function that wraps objects into an envelope.
      */
     public Function<Object, JsonEnvelope> withMetadataFrom(final JsonEnvelope envelope) {
-        return x -> envelopeFrom(buildMetaData(x, envelope.metadata()), objectToJsonValueConverter.convert(x));
+        return x -> envelopeFrom(buildMetaData(x, envelope.metadata()), x == null ? JsonValue.NULL : objectToJsonValueConverter.convert(x));
```

Comment:
Could we do this in the converter? ie, make the converter return NULL if the input is null?